docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.9'
+services:
+  postgres:
+    image: postgres:latest
+    container_name: rms_db
+    ports:
+    - 5432:5432
+    volumes:
+    - pgdata:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: admin
+      POSTGRES_PASSWORD: rmsadmin
+      POSTGRES_DB: rms
+  app:
+    container_name: rms_app
+    build:
+      context: .
+    ports:
+    - 8080:8080
+    depends_on:
+      - postgres
+    environment:
+      SPRING_DATASOURCE_URL: jdbc:postgresql://rms_db:5432/rms
```

Comment:
Można dodać tutaj do ports:
`- 5005:5005`
a do environment:
`JAVA_TOOL_OPTIONS: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005`

Wtedy będzie można debuggować zdalnie aplikację.

Reply:
Można to dodać jak będzie to potrzebne aktualnie chyba tego nie potrzebujemy.